fv3gfs/wrapper/physics_properties.json

```diff
@@ -112,6 +112,13 @@
         "container": "Radtend",
         "dims": ["y", "x"]
     },
+    {
+        "name": "average_albedo_surface",
```

Comment:
What kind of average is this? `time_average_of`? How are averages named in other standard names? Is there a standard name for surface albedo (`surface_albedo`)?

Reply:
Let me investigate if there's cf name or something...it's called that because it averages various near-IR and shortwave/high-low zenith angle albedos.

Reply:
OK nm the standard name is `surface_diffused_shortwave_albedo`. Will change to that